.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  strategy:
+    matrix:
+      features: ["", "--all-features"]
+
+  ubuntu_20_04:
+    name: "ci ubuntu-20.04"
+    runs-on: "ubuntu-20.04"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install GDAL
+        run: |
+          sudo apt-get update
```

Comment:
Alternatively if the setup is slow you can prepare a docker container. We have a couple hosted at https://github.com/georust/docker-images that we use for georust/geo and georust/proj.

It's much faster to run CI this way, especially since we're compiling proj from source. 

The downside is I seem to get into a three hour wrestling match with Docker every 6 months.

Reply:
yeah i guess the docker container is the next step after getting this to work.